avaframe/ana3AIMEC/aimecTools.py

```diff
@@ -134,6 +134,24 @@ def fetchReferenceSimNo(pathDict, cfgSetup):
     return pathDict
 
 
+def computeCellSizeSL(cfgSetup, demHeader):
+    print(cfgSetup['cellSizeSL'])
+    print(cfgSetup['cellSizeSL'].isdigit())
+    if cfgSetup['cellSizeSL'] == '':
+        cellSizeSL = demHeader['cellsize']
+        log.info('cellSizeSL is read from the refference header and is : %.2f m' % cellSizeSL)
+    elif cfgSetup['cellSizeSL'].isdigit():
```

Comment:
I think this doesn't work if there is a decimal point included..

Reply:
I suggest adding a pytest right away testing these things...